File: DictationAssistant/Module1.vb
Module Module1
    Public CiZuZengQiangMuLu As String

    Public Function GetChineseEngineLevel(Name As String) As Integer
        Select Case Name
            Case "VW Lily"
                Return 5
            Case "VW Hui"
                Return 4
            Case "VW Liang"
                Return 3
            Case "VW Wang"
                Return 2
            Case "Girl XiaoKun"
                Return 1
            Case Else
                Return 0
        End Select
    End Function
    Public Function GetEnglishEngineLevel(Name As String) As Integer
        Select Case Name
            Case "Microsoft Hazel Desktop"
                Return 4
            Case "VW Julie"
                Return 3
            Case "VW Paul"
                Return 2
            Case "VW Kate"
                Return 1
            Case Else
                Return 0
        End Select
    End Function
End Module

File: DictationAssistant/Module1.cs
namespace DictationAssistant
{
    static class Module1
    {
        public static string CiZuZengQiangMuLu;

        public static int GetChineseEngineLevel(string Name)
        {
            switch (Name)
            {
                case "VW Lily":
                    {
                        return 5;
                    }

                case "VW Hui":
                    {
                        return 4;
                    }

                case "VW Liang":
                    {
                        return 3;
                    }

                case "VW Wang":
                    {
                        return 2;
                    }

                case "Girl XiaoKun":
                    {
                        return 1;
                    }

                default:
                    {
                        return 0;
                    }
            }
        }
        public static int GetEnglishEngineLevel(string Name)
        {
            switch (Name)
            {
                case "Microsoft Hazel Desktop":
                    {
                        return 4;
                    }

                case "VW Julie":
                    {
                        return 3;
                    }

                case "VW Paul":
                    {
                        return 2;
                    }

                case "VW Kate":
                    {
                        return 1;
                    }

                default:
                    {
                        return 0;
                    }
            }
        }
    }
}
